pkg/scheduler/api/devices/nvidia/gpushare/device_info.go

```diff
@@ -166,47 +166,48 @@ func (gs *GPUDevices) MonitorStatus() string {
 }
 
 func (gs *GPUDevices) AllocateToPod(kubeClient kubernetes.Interface, pod *v1.Pod) error {
+	klog.V(4).Infoln("DeviceSharing:Into AllocateToPod", pod)
```

Comment:
I'm not sure whether the parameter `pod` will be used in the log info.

Reply:
thanks, you are right, now i'm using pod.Name instead